src/keywords.ts

```diff
@@ -8,6 +8,7 @@ export default [
   "asof",
   "backup",
   "between",
+  "boolean",
```

Comment:
boolean is a type, it should be listed in https://github.com/questdb/sql-grammar/blob/master/src/dataTypes.ts

Reply:
`boolean` seems to be there: 

![image](https://user-images.githubusercontent.com/7276403/104315534-02fe8780-54d3-11eb-9cfe-81ade78db282.png)

However it isn't highlighted:

![image](https://user-images.githubusercontent.com/7276403/104315643-2a555480-54d3-11eb-9efb-a569cecc8a5e.png)

Is there problem somewhere else?